### PR TITLE
satellite/gracefulexit: use zap.Stringer instead of zap.String

### PR DESCRIPTION
**Pull request by navillasa**
What/why: using zap.stringer instead of zap.string since it's better

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/gracefulexit/endpoint.go [hunk 1]
```diff
@@ -335,17 +335,17 @@ func (endpoint *Endpoint) processIncomplete(ctx context.Context, stream processS
...
-	endpoint.log.Debug("found new node for piece transfer.", zap.String("original node ID", nodeID.String()), zap.String("replacement node ID", newNode.Id.String()),
+	endpoint.log.Debug("found new node for piece transfer", zap.Stringer("original node ID", nodeID), zap.Stringer("replacement node ID", newNode.Id),
 		zap.ByteString("path", incomplete.Path), zap.Int32("piece num", incomplete.PieceNum))
 
 	pieceID := remote.RootPieceId.Derive(nodeID, incomplete.PieceNum)
 
 	parts := storj.SplitPath(storj.Path(incomplete.Path))
 	if len(parts) < 2 {
-		return Error.New("invalid path for %v %v.", zap.String("node ID", incomplete.NodeID.String()), zap.String("pieceID", pieceID.String()))
+		return Error.New("invalid path for node ID %v, piece ID %v", zap.Stringer("node ID", incomplete.NodeID), zap.Stringer("piece ID", pieceID))
```
Comment (egonelbre) [inline satellite/gracefulexit/endpoint.go]: Only logging should use zap values. This should be:

```
Error.New("invalid path for node ID %v, piece ID %v", incomplete.NodeID, pieceID)
```

There probably are other places that have this mistake.

Comment (navillasa) [reply satellite/gracefulexit/endpoint.go]: oh oops, I'll fix. thanks for catching.

Comment (navillasa) [reply satellite/gracefulexit/endpoint.go]: @egonelbre should the errors use nodeID.String() or just nodeID?

Comment (egonelbre) [reply satellite/gracefulexit/endpoint.go]: It's the same thing. If you use `%v` or `%s` then Go will first try to use `.String()` method.

**Pull request merged**